digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/OracleAssetDao.scala

```diff
@@ -76,4 +76,18 @@ class OracleAssetDao {
     sql"""select ID, ASSET_TYPE_ID from ASSET where ID in (#${ids.mkString(",")})""".as[(Long, Int)].list
   }
 
+
+  def getVerifiableAssetTypes: Seq[String] = {
+    val assetTypes = sql"""select name from asset_type where verifiable = 1""".as[(String)].list
+    assetTypes
+  }
+
+  def getMunicipalitiesNameByCode(codes: Set[Int]): List[(Int, String)] = {
```

Comment:
duplicate code.  On `MunicipalityDao.scala` is an equal method with different name :  `getMunicipalitiesNameandIdByCode`